ext/ddtrace.h

```diff
@@ -75,7 +75,13 @@ typedef struct {
 
 typedef struct {
     uint64_t low;
-    uint64_t high;
+    union {
+        uint64_t high;
+        struct {
+            uint32_t padding; // zeroes
+            uint32_t time;
+        };
```

Comment:
Based on your commit comment, this is endian specific, yes?

Reply:
Yes it does, I'll just do it (though we only really target little endian).